### ISSUE TITLE
New volume label will return to ISO image label after i finished a quick format only

### ISSUE THREAD

**Issue by Sopor-** [pbatard/rufus#490]
If i load an image file the 'New volume label' will change to the label i have on the image file. If i instead unchecked all except 'Quick format' i still have the label from the image file. I then change it because i want another name on my USB-device. I then start the quick format and when it's done it now change back to the label on the image file. So i was thinking, have it used my label or have it used the label from the image file?
When i put in a USB-device i can see the label on the device and when i load an image file it will change to that label but i suggest that it should only do that if the 'Create a bootable disk using' is selected and when i only quick format a device i have it unchecked so when the format is finished it should show the label of the USB-device instead of the image label.

Short version: Even if the image file is loaded it should only show the label from the image file if 'Create a bootable disk using' is selected. If i unselect it, show the USB-device label instead.

**Comment by pbatard** [comment]
I'll see what I can do, but since this is a minor UI __preference__, and UI changes are always a major pain in the ass to handle (because changing one minor visible feature, usually impacts a whole bunch of user behaviour paths that are already coded in the app) I will keep this a __super-low__ priority.
